### PR TITLE
Several UI Tweaks - categories, but mostly datasets page

### PR DESCRIPTION
**Pull request by BryanQuigley**
1. Drop old Add-categories link (not shown)

2. Spacing changes 
* UI: Sidebar: Increase spacing between Categories and Organizations
* UI: Results: Increase spacing to make list more legible. I just added top spacing to datasets to the separator is more in the middle.

image

3.  The original suggestion was Change count pill for selected category and X button to white background but I decided to use Red fox X and a lighter blue. I am open to changing it to another scheme  - or all gray was another option.

image

### CODE REVIEW

File: scripts/src/templates/list-group-item.js [hunk 1]
```diff
@@ -2,8 +2,9 @@ export default (data) => (
 `<a href="${data.url}" class="list-group-item${data.selected ? ' active ' : ''} list-group-item-action d-flex justify-content-between align-items-center">
   ${data.title}
   <span>
-    ${data.selected ? '<span class="badge bg-secondary rounded-pill"><i class="fa fa-times"></i></span>' : ''}
-    <span class="badge bg-secondary rounded-pill">${data.count}</span>
+    ${data.selected ? '<span class="badge bg-danger rounded-pill"><i class="fa fa-times"></i></span>' : ''}
+    ${data.selected ? '<span class="badge bg-info rounded-pill">' : '<span class="badge bg-primary rounded-pill">'}
```
Comment (timwis) [inline scripts/src/templates/list-group-item.js:6]: I think `bg-info` is meant to be paired with `text-dark` to maintain contrast, but can I actually suggest `bg-light text-dark` for both badges? I think that provides sufficient contrast against the selected blue/primary, without drawing too much attention.

Comment (timwis) [reply scripts/src/templates/list-group-item.js:6]: Oh, and can we update it in `datasets.html` as well, for the server-rendered version?

File: categories.html [hunk 2]
```diff
@@ -5,8 +5,6 @@
 ---
 {% include breadcrumbs.html %}
 
-<a href="{{ site.baseurl }}/add-category/" class="btn btn-outline-secondary float-end admin-only" data-hook="add-category-btn"><i class="fa fa-plus"></i> Add category</a>
```
Comment (timwis) [inline categories.html:8]: Thanks - completely missed that!

**Review by timwis — CHANGES_REQUESTED** [review]
Looks like good changes! Just left a couple minor requests on the badges

File: scripts/src/templates/list-group-item.js (hunk 1 above)
Comment (BryanQuigley) [reply scripts/src/templates/list-group-item.js:6]: That's much nicer! switching to proposed colors.

**Pull request merged**